runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java

```diff
@@ -172,6 +172,16 @@ public void onChatMessage(ChatMessage chatMessage)
 					notifier.notify(Text.removeTags(chatMessage.getName()) + ": " + chatMessage.getMessage());
 				}
 				break;
+			case PRIVATECHATOUT:
+				return;
+			case PUBLICCHAT:
+			case FRIENDSCHAT:
+			case AUTOTYPER:
+				if (client.getLocalPlayer() != null && chatMessage.getName().equals(client.getLocalPlayer().getName()))
```

Comment:
This fails for players with spaces in their name. `Player#getName()` appears to return a normal space, while `ChatMessage#getName()` uses a nbsp. It's likely safest to assume both strings must be run through `Text#toJagexName()`.

The test case covering this should be updated as well.